### DISCUSSION TITLE
turn camera

### DISCUSSION THREAD

**Discussion by barraviera** [pedroSG94/RootEncoder#1130] [Q&A]
is it possible to turn the camera while still transmitting?

**Comment by pedroSG94** [comment]
If you means change from front camera to back camera. Yes, you have switchCamera method to do it. You can check it in my code examples:
https://github.com/pedroSG94/rtmp-rtsp-stream-client-java/blob/master/app/src/main/java/com/pedro/rtpstreamer/defaultexample/ExampleRtmpActivity.java#L166

**Reply by barraviera** [reply]
I am using backgroundexemple. I would like to change the camera without stopping the flow, is that possible?

**Reply by pedroSG94** [reply]
> I am using backgroundexemple. I would like to change the camera without stopping the flow, is that possible?

As I told you try using switchCamera method to do it.
